react/features/participants-pane/components/native/ContextMenuLobbyParticipantReject.js

```diff
@@ -35,7 +35,6 @@ const ContextMenuLobbyParticipantReject = ({ participant: p }: Props) => {
         <View
             style = { styles.contextMenuItemSectionAvatar }>
             <Avatar
-                className = 'participant-avatar'
```

Comment:
Why is this change necessary?

Reply:
Because we don't use classNames on native. This was added in a separate commit.

Reply:
👍